### ISSUE TITLE
[Skype] VideoCard does not play on iOS app

### ISSUE THREAD

**Issue by nwhitmont** [Microsoft/BotBuilder#2709]
Bot Framework VideoCard does not play video on Skype mobile app for iOS.

![img_9896](https://cloud.githubusercontent.com/assets/3075047/25766545/093176d2-31a8-11e7-970b-8e0eafe8317a.PNG)

**Comment by nwhitmont** [comment]
Sample bot used to test Skype mobile: https://github.com/Microsoft/BotBuilder-Samples/tree/master/Node/cards-RichCards

**Comment by nwhitmont** [comment]
@nickcordrey any updates on this Skype mobile bug?

**Comment by nwhitmont** [comment]
@nickcordrey Any updates on VideoCard does not play video in Skype mobile iOS?

**Comment by aery-aery** [comment]
@nwhitmont The problem seems to be caused by using http url for video. Using http violates Apple’s App Transport Security rules (https://developer.apple.com/news/?id=12212016b). The same thing happens in #2708.

**Comment by nwhitmont** [comment]
@aery-aery I updated all my media URLs to `https` & redeployed the bot, but the video card still doesn't work. You can verify my [test bot code here](https://github.com/nwhitmont/botframework-skype-support-dev/blob/master/server.js#L165).

**Comment by aery-aery** [comment]
@nwhitmont can you describe what do you see when you try to play the video?
I've tried to reproduce it with rest api by posting this activity
```{
  "type": "message",
  "attachments": [
    {
      "contentType": "application/vnd.microsoft.card.video",
      "content": {
        "title": "Big Buck Bunny",
        "subtitle": "by the Blender Institute",
        "text": "Big Buck Bunny (code-named Peach) is a short computer-animated comedy film by the Blender Institute, part of the Blender Foundation. Like the foundation's previous film Elephants Dream, the film was made using Blender, a free software application for animation made by the same foundation. It was released as an open-source film under Creative Commons License Attribution 3.0.",
        "image": {
          "url": "https://upload.wikimedia.org/wikipedia/commons/thumb/c/c5/Big_buck_bunny_poster_big.jpg/220px-Big_buck_bunny_poster_big.jpg"
        },
        "media": [
          {
            "url": "https://download.blender.org/peach/bigbuckbunny_movies/BigBuckBunny_320x180.mp4"
          }
        ],
        "buttons": [
          {
            "type": "openUrl",
            "value": "https://peach.blender.org/",
            "title": "Learn More"
          }
        ]
      }
    }
  ]
  }
}```

And it seems to work for me

**Comment by nwhitmont** [comment]
@aery-aery I see the triangle with exclamation mark as shown in the image in my original message. My Node.js SDK Bot is hosted on Azure App Service on Linux.

**Comment by nwhitmont** [comment]
this appears to be fixed in Skype iOS v6.35.0.54
